parachain/src/chain_spec/local.rs

```diff
@@ -1,23 +1,20 @@
 use cumulus_primitives_core::ParaId;
 use hex_literal::hex;
 use local_runtime::{
-	AccountId, EthereumHeader,
```

Comment:
Woah :astonished:

Please revert. You're reformatting the entire file. Its also not the correct formatting style. We use tabs instead of spaces for Rust code.

I suspect you've got format-on-save enabled in VS Code for both Rust and Solidity code. While that's actually good practice, we need to agree on a common formatting style, and then make sure our editors all use the same config. Otherwise we're going to have lots of whitespace/formatting churn throughout our commits that make it harder to use `git blame` for historical analysis.

So I suggest you disable format-on-save for now.